tests/integration/api/test_api.py

```diff
@@ -753,5 +753,152 @@ def test_host_muting(self):
+        # Save the html to check against replaced var get
+        html = created_graph["html"]
+        # Save the embed_id into a variable and get it again
+        embed_id = created_graph["embed_id"]
+        response_graph = dog.Embed.get(embed_id, var="asdfasdfasdf")
+        # Check the graph has the same embed_id and the template_var is added to the url
+        assert "embed_id" in response_graph
+        assert response_graph["embed_id"] == embed_id
+        assert len(response_graph["html"]) > len(html)
+
+    @attr('embed')
+    def test_enable_embed(self):
+        # Create a graph that we can try getting
+        graph_def = {
+            "viz": "toplist",
+            "requests": [{
+                "q": "top(system.disk.free{$var} by {device}, 10, 'mean', 'desc')",
+                "style": {
+                    "palette": "dog_classic"
+                },
+                "conditional_formats": [{
+                    "palette": "red",
+                    "comparator": ">",
+                    "value": 50000000000
+                }, {
+                    "palette": "green",
+                    "comparator": ">",
+                    "value": 30000000000
+                }]
+            }]
+        }
+        timeframe = "1_hour"
+        size = "medium"
+        legend = "no"
+        graph_json = json.dumps(graph_def)
+        created_graph = dog.Embed.create(graph_json=graph_json, timeframe=timeframe, size=size, legend=legend)
+        # Save the embed_id into a variable and enable it again
+        embed_id = created_graph["embed_id"]
+        result = dog.Embed.enable(embed_id)
+        # Check that the graph is enabled again
+        assert "success" in result
+
+    @attr('embed')
+    def test_revoke_embed(self):
+        # Create a graph that we can try getting
+        graph_def = {
+            "viz": "toplist",
+            "requests": [{
+                "q": "top(system.disk.free{$var} by {device}, 10, 'mean', 'desc')",
+                "style": {
+                    "palette": "dog_classic"
+                },
+                "conditional_formats": [{
+                    "palette": "red",
+                    "comparator": ">",
+                    "value": 50000000000
+                }, {
+                    "palette": "green",
+                    "comparator": ">",
+                    "value": 30000000000
+                }]
+            }]
+        }
+        timeframe = "1_hour"
+        size = "medium"
+        legend = "no"
+        graph_json = json.dumps(graph_def)
+        created_graph = dog.Embed.create(graph_json=graph_json, timeframe=timeframe, size=size, legend=legend)
+        # Save the embed_id into a variable and enable it again
+        embed_id = created_graph["embed_id"]
+        result = dog.Embed.revoke(embed_id)
+        # Check embed is revoked and that we can't get it again
+        assert "success" in result
+        try:
```

Comment:
I know we us this pattern a lot, but can we switch to `assertRaises` instead (c.f. https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises).